x/virtualgroup/keeper/msg_server.go

```diff
@@ -493,23 +514,141 @@ func (k msgServer) CompleteStorageProviderExit(goCtx context.Context, msg *types
 		return nil, err
 	}
 
-	// send back the total deposit
-	coins := sdk.NewCoins(sdk.NewCoin(k.spKeeper.DepositDenomForSP(ctx), sp.TotalDeposit))
-	err = k.bankKeeper.SendCoinsFromModuleToAccount(ctx, sptypes.ModuleName, sdk.MustAccAddressFromHex(sp.FundingAddress), coins)
+	var forcedExit bool
+	if sp.Status == sptypes.STATUS_GRACEFUL_EXITING {
+		// send back the total deposit
+		coins := sdk.NewCoins(sdk.NewCoin(k.spKeeper.DepositDenomForSP(ctx), sp.TotalDeposit))
+		err = k.bankKeeper.SendCoinsFromModuleToAccount(ctx, sptypes.ModuleName, sdk.MustAccAddressFromHex(sp.FundingAddress), coins)
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		forcedExit = true
+		coins := sdk.NewCoins(sdk.NewCoin(k.spKeeper.DepositDenomForSP(ctx), sp.TotalDeposit))
+		err = k.bankKeeper.SendCoinsFromModuleToModule(ctx, sptypes.ModuleName, govtypes.ModuleName, coins)
```

Comment:
Why send the deposit to gov module?
I think gov module is different from payment governance account.

Reply:
fixed, The deposit will be transfer to the payment module gov addr stream record related bank account